### ISSUE TITLE
Delete or deprecate this repo

### ISSUE THREAD

**Issue by tlvince** [fielded/pouchdb-alldocs-replicate#2]
New work is being done in https://github.com/fielded/pouchdb-onerequest-replicate. Probably we can delete this repo since we're the only ones using it. Otherwise, deprecate it.

**Comment by kdoran** [comment]
@tlvince yep, left it up to avoid breaking any npm installs going on but once we switch I'll delete this, thanks for ticketing it 👍
